docs/reference/cli/bc_notify_subscribe.md

```
@@ -0,0 +1,47 @@
+## bc notify subscribe
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)